politeiawww/user/cockroachdb/log.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2013-2015 The btcsuite developers
+// Copyright (c) 2013-2021 The btcsuite developers
```

Comment:
We didn't add any new code to this file so we don't need to change the copyright.  Also, if we did add new code we would add a `// Copyright (c) 2021 The Decred developers` line and not update the `btcsuite` line.  Different teams.

Reply:
oppss